components/layout/inset-content/inset-content.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="section--bg-white">
+  <div class="inset-content">
+    <h2 class="inset-content__headline">Why partner with us?</h2>
+
+    <h4 class="inset-content__title">Dedicated consultants</h4>
```

Comment:
Ideally, these would be h3's so that we aren't skipping a level in the outline.

Separate, do we like the header+paragraph structure here more than a definition list for purposes of semantics? I'm not exactly swinging for one or the other, just feels like a question we should ask.